superbench/benchmarks/micro_benchmarks/disk_performance.py

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the Disk Performance benchmarks."""
+
+from pathlib import Path
+import json
+import os
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, ReturnCode
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class DiskPerformance(MicroBenchmarkWithInvoke):
+    """The KernelLaunch overhead benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'fio'
+
+        self._io_patterns = ['seq', 'rand']
```

Comment:
For private member, we prefer to use two underline at the beginning. Please refer
https://github.com/microsoft/superbenchmark/blob/702fb1eb3713b2a8c8d9070f7819bba7bab39be4/superbench/benchmarks/micro_benchmarks/sharding_matmul.py#L43

Reply:
Fixed.